### PR TITLE
Update Management.Compute from 9.1.0 to 9.2.0

### PR DESCRIPTION
**Pull request by hyonholee**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@hyonholee__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (hylee). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/AutomationManagement/Microsoft.WindowsAzure.Management.Automation.nuspec [hunk 1]
```diff
@@ -23,7 +23,7 @@
       </group>
     </references>
     <dependencies>
-      <dependency id="Microsoft.Azure.Common" version="[2.0.3,3.0)" />
+      <dependency id="Microsoft.Azure.Common" version="[2.0.2,3.0)" />
```
Comment (ogail) [inline src/AutomationManagement/Microsoft.WindowsAzure.Management.Automation.nuspec]: Why the dependency is down graded?

Comment (markcowl) [inline src/AutomationManagement/Microsoft.WindowsAzure.Management.Automation.nuspec]: This actually won't work - the msi installer in PowerSehll will not allow a downgrade

**Comment by yugangw-msft** [comment]
let us merge it after https://github.com/Azure/azure-sdk-for-net/pull/978 gets in first

**Pull request merged**